azimuth/utils/utterance.py

```diff
@@ -13,5 +13,5 @@ def clean_utterance(utterance: str) -> str:
         utterance_clean: Cleaned utterance.
 
     """
-    utterance_clean = utterance.replace("’", "'").lower()  # Quote->apostrophe (unicode 8217 -> 39)
+    utterance_clean = utterance.replace("’", "'").replace("`", "'").lower()
```

Comment:
I think it's fine to repeat `replace(..., "'")` twice, but if it bothers you, or if you suspect we might need to add more, we could replace them all at one call using a regular expression.
```py
import re

re_irregular_apostrophes = re.compile("[’`]")
```
```suggestion
    utterance_clean = re_irregular_apostrophes.sub("'", utterance).lower()
```

Reply:
Also, the name `re_irregular_apostrophes` serves as a comment. Maybe `re_unusual_apostrophes` is better? Or `re_atypical_apostrophes`? 🤷‍♂️

Reply:
Oh cool!! I really need to learn about regular expressions..